run_unittests.py

```diff
@@ -3604,13 +3604,14 @@ def test_native_dep_pkgconfig(self):
                 pkgconfig = r'{0}'
 
                 [properties]
+                pkg_config_libdir = [r'{1}']
 
                 [host_machine]
                 system = 'linux'
                 cpu_family = 'arm'
                 cpu = 'armv7'
                 endian = 'little'
-                '''.format(os.path.join(testdir, 'cross_pkgconfig.py'))))
+                '''.format(os.path.join(testdir, 'cross_pkgconfig.py'), os.path.join(testdir, 'cross_pkgconfig'))))
```

Comment:
You're not testing the case where we have a pkg-config wrapper instead of using pkg_config_libdir any more. That use-case is still valid and we have to ensure it's still working because some toolchains (mingw on linux distro) ship with that wrapper.

So you should not change that test, but add a similar test in addition to it. Ideally by sharing some code if you can refactor a bit.